docs/docker.md

```diff
@@ -0,0 +1,40 @@
+# Docker
+
+Docker file naming convention.
```

Comment:
Please format the style of this section:

Docker file naming convention: `Dockerfile.{type}.{extra tags}.{baseimage}`
- **_type_**: test, prod ...
- **_extra tags_**: sqlite, clang7 ...
- **_baseimage_**: xenial, binoic, archlinux ...

Reply:
This line still looks unformatted to me:
![image](https://user-images.githubusercontent.com/6695818/49073756-6b2a7a80-f233-11e8-8716-c900e2140e79.png)
Please format it as I recommended in my above comment.